app/src/main/res/layout/activity_wave_generator_new.xml

```diff
@@ -0,0 +1,1062 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Create multiple layout files and stack them in this file instead of putting everything in one file.